projects.md

```diff
@@ -45,40 +47,46 @@ projects:
         link: https://www.figma.com/community/file/916680391812923706/Bitcoin-Wallet-UI-Kit-(work-in-progress)
       - name: Github
         link: https://github.com/GBKS/bitcoin-wallet-ui-kit
-collaborations:
-  - name: Specter
-    description: A watch-only coordinator for multi-signature and single-key Bitcoin wallets.
+
+  - name: Wallet design improvement project
+    description: Improving Lightning network wallets through applying the design guide, design thinking and problem solving.
     image:
-      url: /assets/images/contribute/specter.png
+      url: /assets/images/contribute/wallt-design-imporvement.png
       width: 75
       height: 75
     links:
       - name: Site
-        link: https://specter.solutions/
-      - name: Collaboration
-        link: https://github.com/BitcoinDesign/Meta/issues/69
-  - name: Zeus
-    description: Manage your Bitcoin and Lightning node on the go.
+        link: https://github.com/BitcoinDesign/Guide/issues/493
```

Comment:
```suggestion
        link: https://github.com/BitcoinDesign/Meta/issues/244
```

@pavlenex set up a collaboration issue we can link to that will be kept up-to-date.